oap-logstream/pom.xml

```diff
@@ -25,11 +25,47 @@
                 </exclusion>
             </exclusions>
         </dependency>
-       <dependency>
+        <dependency>
+            <groupId>oap</groupId>
+            <artifactId>oap-template</artifactId>
```

Comment:
It is also added to `oap-logstream-data`. So should it be declared in root POM onky once?

Reply:
BinaryObjectLogger depends on TemplateEngine